secret_number.js

```diff
@@ -1,4 +1,15 @@
 'use strict';
 module.exports = function() {
-  
-};
\ No newline at end of file
+  let secretNumber = (Math.random()*1000000 + 1);
```

Comment:
This solution will never resolve to 0, which should be a possibility.
Also, secret numbers should only be integers.